test/test_integration.py

```diff
@@ -27,8 +26,11 @@ def test_start_beebop():
     session = requests.Session()
     session.verify = False
     session.trust_env = False
-    os.environ['CURL_CA_BUNDLE'] = ""
-    res = session.get("https://localhost/api/", verify=False)
+
+    # Give services time to fully initialize
+    time.sleep(5)
```

Comment:
oh dear! wonder why that's crept in...

Reply:
yeah i tried so many things and this was the only things let It actually consistently pass without have SSL / connection errors